zuul.d/jobs.yaml

```diff
@@ -108,3 +108,12 @@
       nodes:
         - name: builder
           label: ubuntu-xenial-builder
+
+- job:
+    name: contrail-vnc-docker-provision-centos74
+    voting: False
+    run: playbooks/docker/centos74-provision
+    nodeset:
+      nodes:
+        - name: builder
+          label: centos-7.4-builder
```

Comment:
This node "type" is probably way too large for provision/systest jobs (8vcpu/16GB sounds excessive)

Reply:
Do we have something more suitable right now?